### PR TITLE
DOC: Improve documentation for `jnp.fliplr` and `jnp.flipud`

### PR DESCRIPTION
**Pull request by rajasekharporeddy**
Part of #21461

### CODE REVIEW

File: jax/_src/numpy/lax_numpy.py [hunk 1]
```diff
@@ -729,14 +729,54 @@ def _flip(m: Array, axis: int | tuple[int, ...] | None = None) -> Array:
...
 
 
-@util.implements(np.fliplr, lax_description=_ARRAY_VIEW_DOC)
 def fliplr(m: ArrayLike) -> Array:
+  """Reverse the order of elements of an array along axis 1.
+
+  JAX implementation of :func:`numpy.fliplr`.
+
+  Args:
+    m: Array.
```
Comment (jakevdp) [inline jax/_src/numpy/lax_numpy.py:738]: array with at least two dimensions

File: jax/_src/numpy/lax_numpy.py [hunk 2]
```diff
@@ -729,14 +729,54 @@ def _flip(m: Array, axis: int | tuple[int, ...] | None = None) -> Array:
...
 
 
-@util.implements(np.flipud, lax_description=_ARRAY_VIEW_DOC)
 def flipud(m: ArrayLike) -> Array:
+  """Reverse the order of elements of an array along axis 0.
+
+  JAX implementation of :func:`numpy.flipud`.
+
+  Args:
+    m: Array.
```
Comment (jakevdp) [inline jax/_src/numpy/lax_numpy.py:764]: Array with at least one dimension

File: jax/_src/numpy/lax_numpy.py (hunk 1 above)
Comment (rajasekharporeddy) [reply jax/_src/numpy/lax_numpy.py]: Done

File: jax/_src/numpy/lax_numpy.py (hunk 2 above)
Comment (rajasekharporeddy) [reply jax/_src/numpy/lax_numpy.py]: Done

File: jax/_src/numpy/lax_numpy.py [hunk 3]
```diff
@@ -729,14 +729,54 @@ def _flip(m: Array, axis: int | tuple[int, ...] | None = None) -> Array:
...
 
 
-@util.implements(np.fliplr, lax_description=_ARRAY_VIEW_DOC)
 def fliplr(m: ArrayLike) -> Array:
+  """Reverse the order of elements of an array along axis 1.
+
+  JAX implementation of :func:`numpy.fliplr`.
+
+  Args:
+    m: Array with atleast two dimenssions.
```
Comment (jakevdp) [inline jax/_src/numpy/lax_numpy.py:738]:
```suggestion
    m: Array with at least two dimensions.
```

File: jax/_src/numpy/lax_numpy.py [hunk 4]
```diff
@@ -729,14 +729,54 @@ def _flip(m: Array, axis: int | tuple[int, ...] | None = None) -> Array:
...
 
 
-@util.implements(np.flipud, lax_description=_ARRAY_VIEW_DOC)
 def flipud(m: ArrayLike) -> Array:
+  """Reverse the order of elements of an array along axis 0.
+
+  JAX implementation of :func:`numpy.flipud`.
+
+  Args:
+    m: Array with atleast one dimension.
```
Comment (jakevdp) [inline jax/_src/numpy/lax_numpy.py:764]:
```suggestion
    m: Array with at least one dimension.
```

**Review by jakevdp — CHANGES_REQUESTED** [review]

File: jax/_src/numpy/lax_numpy.py (hunk 3 above)
Comment (rajasekharporeddy) [reply jax/_src/numpy/lax_numpy.py]: Done

File: jax/_src/numpy/lax_numpy.py (hunk 4 above)
Comment (rajasekharporeddy) [reply jax/_src/numpy/lax_numpy.py]: Done

**Review by jakevdp — APPROVED** [review]
Thanks!

**Pull request merged**